h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -454,7 +453,15 @@ def standalone_test(test):
     h2o.log_and_echo("STARTING TEST")
     h2o.log_and_echo("")
     h2o.log_and_echo("------------------------------------------------------------")
-    test()
+    if issubclass(test, TestCase):
+        test_instance = test()
+        callable_methods = [method for method in dir(test) if callable(getattr(test, method))]
+        for method in (method for method in callable_methods if method.startswith('test_')) :
```

Comment:
Do you think we should allow people to define multiple test methods in one test case? How will our framework handle test failures? Do we get a fine-grained notification of what went wrong?

Should we clean-up after each test method? (see lines 450-456)? What do you think?

Reply:
The whole class would fail and further tests would not be executed. The rest of pipeline would react the same way as it did before.